synopsys-task/test/unit/synopsys-task/tools-parameter.spec.ts

```diff
@@ -108,6 +108,50 @@ describe("Synopsys Tools Parameter test", () => {
             expect(formattedCommand).contains('--input '.concat(polarisStateFile));
         });
 
+        it('should success for polaris command formation with Arbitrary arguments', async function () {
```

Comment:
Shall we unset the defined property as well, so that the same variable can be used in other negative scenario test cases(if we are planning to write)

Reply:
done.